source/faq.txt

```diff
@@ -4,12 +4,165 @@ FAQ
 
 .. default-domain:: mongodb
 
-Frequently Asked Questions
-
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-Lorem ipsum.
+General
+-------
+
+Why are there two versions of the classes? e.g. ``Mongo`` and ``MongoClient``?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``MongoDatabase``, ``MongoClient``, ``MongoCollection`` are the
+newer generation of classes and have better defaults for production use.
```

Comment:
nitpick: Remove 'The' at the beginning, or add 'classes' after the 3 classes listed here and then remove 'classes' later in the sentence